packages/zoe/test/unitTests/contracts/test-sellTickets.js

```diff
@@ -0,0 +1,570 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
new-SES: new test files that need to `makeZoe()` will need to do `import '@agoric/install-ses'` before all other imports , change `makeZoe` and `install` calls as before